apis/tempo/v1alpha1/microservices_webhook.go

```diff
@@ -11,28 +15,50 @@ import (
 )
 
 var (
-	zeroQuantity  = resource.MustParse("0Gi")
-	tenGBQuantity = resource.MustParse("10Gi")
+	zeroQuantity                = resource.MustParse("0Gi")
+	tenGBQuantity               = resource.MustParse("10Gi")
+	ErrNoDefaultTempoImage      = errors.New("please specify a tempo image in the CR or in the operator configuration")
+	ErrNoDefaultTempoQueryImage = errors.New("please specify a tempo-query image in the CR or in the operator configuration")
```

Comment:
```suggestion
	errNoDefaultTempoImage      = errors.New("please specify a tempo image in the CR or in the operator configuration")
	errNoDefaultTempoQueryImage = errors.New("please specify a tempo-query image in the CR or in the operator configuration")
```

We do not use this errors somewhere else, right? If so, i would recommend to keep them private.

Reply:
Good idea, I've updated the PR.